noxfile.py

```diff
@@ -614,6 +615,10 @@ def conda_test(session):
     # for troubleshooting purposes.
     session.run("mamba", "list")
 
+    subprocess.run(
```

Comment:
```suggestion

    # Using subprocess.run() instead of session.run() because
    # session.run() does not correctly handle the pip check command.
    subprocess.run(
```